src/framework/components/router/core.clj

```diff
@@ -7,12 +7,23 @@
   [db]
   component/Lifecycle
   (stop [this] this)
-  (start
-    [this]
-    (assoc this :ring-router (ring/router (concat
-                                            (:routes (:specific-component this)) ; adds routes specific to some component
-                                            (:custom-routes this))))))
+  (start [this]
+         (assoc this :ring-router (ring/router (concat
+                                                 (:routes (:specific-component this)) ; adds routes specific to some component
+                                                 (:custom-routes this))))))
 
 (defn make-router
```

Comment:
We can mark it as Deprecated with different way, I think.

Reply:
How? :)

Reply:
With Meta data, of course. :)